lib/providers/services/http_service.dart

```diff
@@ -559,7 +560,51 @@ class HttpService {
       return false;
     }
   }
+
+  Future<VoteResult> getVote({proposalId: int}) async {
+
+    String url = "$baseURL/v1/chain/get_table_rows";
+
+    var request = '''{
+      "json": true,
+      "code": "funds.seeds",
+      "scope": "$proposalId",
+      "table": "votes",
+      "table_key": "",
+      "lower_bound": "$userAccount",
+      "upper_bound": "$userAccount",
+      "limit": 10,
+      "key_type": "",
+      "index_position": "",
+      "encode_type": "dec",
+      "reverse": false,
+      "show_payer": false
+    }''';
+
+    Map<String, String> headers = {"Content-type": "application/json"};
+
+    Response res = await post(url, headers: headers, body: request);
+    if (res.statusCode == 200) {
+      Map<String, dynamic> body = jsonDecode(res.body);
+
+print("body: $body");
```

Comment:
For future reference. 
Having print in production code seems to be a bad practice in flutter, since Any print statement can be potentially seen by any user of the app, which can be a problem if we print sensitive information.

Reply:
Well definitely - they're all removed when going into production

Only exception - I like to leave print statements for error cases, that makes it easier to debug things when they go wrong. 

But for any non-error cases - no print statements

Reply:
👍

Reply:
This has been removed FWIW